tests/sentry/receivers/test_releases.py

```diff
@@ -68,7 +68,7 @@ def test_updating_commit(self):
         repo = Repository.objects.create(name="example", organization_id=self.group.organization.id)
 
         commit = Commit.objects.create(
-            key=sha1(uuid4().hex).hexdigest(),
+            key=sha1(uuid4().hex.encode("utf-8")).hexdigest(),
```

Comment:
Not sure if this would break tests but we can also do `uuid4().bytes` directly?

Reply:
> The UUID as a 16-byte string (containing the six integer fields in big-endian byte order).

Looks like it would give us different values.

Reply:
Bytes unfortunately produces a different result
Py2
```
In [24]: the_uuid = UUID('f193cef5-a823-4528-9004-85b82b88a0f3')

In [25]: sha1(the_uuid.hex).hexdigest()
Out[25]: 'f1bdcc6907fd350f98b3292d79ae7492a4b4b898'
```

Py3
```
In [61]: the_uuid = UUID('f193cef5-a823-4528-9004-85b82b88a0f3')

In [62]: sha1(the_uuid.hex.encode("utf-8")).hexdigest()
Out[62]: 'f1bdcc6907fd350f98b3292d79ae7492a4b4b898'

In [63]: sha1(the_uuid.bytes).hexdigest()
Out[63]: '5061248744f06db5a934a04d87cb8a9890a9a8ca'
```

Reply:
Yeah, I wasn't clear in my comment again :D I thought these are just test SHAs and wouldn't harm if they were different (that's what I referred to when I said "if this would break tests"). Anyway, I'm assuming these are fixtures and the generated SHAs should not change, hence my suggestion does not apply.